examples/2d/mesh2d_vertex_color_texture.rs

```diff
@@ -0,0 +1,39 @@
+//! Shows how to render a polygonal [`Mesh`], generated from a [`Quad`] primitive, in a 2D scene.
+
+use bevy::{prelude::*, sprite::MaterialMesh2dBundle};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+    asset_server: Res<AssetServer>,
+) {
+    // Load the Bevy logo
+    let texture_handle = asset_server.load("branding/banner.png");
+    // Build a default quad mesh
+    let mut mesh = Mesh::from(shape::Quad::default());
+    // Build vertex colors for the quad
+    let vertex_colors: Vec<[f32; 4]> = vec![
```

Comment:
This matrix is a little inscrutable for beginners. What do these numbers mean? 

I can guess that they're RGBA, with each row representing another vertex, but beginners won't be able to guess that.

Reply:
I agree, so I've replaced them with a much more readable `Color::RED.as_rgba_f32()`. Thanks!